Makefile

```
@@ -144,3 +144,7 @@ lint:
 .PHONY: format
 format:
 	./scripts/format.sh
+
+.PHONY: deploy-dev-stack
+deploy-dev-stack:
+	./scripts/deploy-dev-stack.sh
```

```suggestion
	./scripts/deploy-dev-stack.sh $1
```

Should we pass through the first arg, since looks like the script will take this and assign it to  STACK_NAME if not set, so folks can do something like `make deploy-dev-stack pulumi/cnunciato`

---

Yeah I'd love it if that worked, but I don't believe it's possible to pass an argument to a make target like that. (I've never been able to make it work, at least.)

---

ahhh ok, hmm... I was so sure I have gotten this to work before in the past. I could be misremembering though.